samza-yarn/src/test/java/org/apache/samza/job/yarn/TestFileSystemImplConfig.java

```diff
@@ -45,12 +47,6 @@ public void testFileSystemImplConfigSuccess() {
     assertEquals(2, manager.getSchemes().size());
     assertEquals("http", manager.getSchemes().get(0));
     assertEquals("myscheme", manager.getSchemes().get(1));
-
-    assertEquals("fs.http.impl", manager.getFsImplKey("http"));
-    assertEquals("fs.myscheme.impl", manager.getFsImplKey("myscheme"));
-
-    assertEquals("org.apache.samza.HttpFileSystem", manager.getFsImplClassName("http"));
```

Comment:
Because we removed these two functions, and introduced another function to get the key-value together as a sub-config.